components/jenkins/main.tf

```
@@ -28,6 +28,7 @@ resource "azurerm_managed_disk" "disk" {
   create_option        = "Empty"
   disk_size_gb         = "1024"
   tags                 = module.tags.common_tags
+  zone                 = ["1"]
```

will this try re-create it though? (might be ok)
```suggestion
  zones                = ["1"]
```